package.json

```diff
@@ -98,7 +98,7 @@
     "@babel/preset-env": "7.16.11",
     "@babel/preset-react": "7.16.7",
     "@babel/preset-typescript": "7.16.7",
-    "@commitlint/cli": "16.2.1",
+    "@commitlint/cli": "16.2.3",
```

Comment:
Bad rebase ?

Reply:
it has been made automatically after rebase, so what you mean by bad rebase ? I have to revert the change on package.json

Reply:
Yes, I don't think you are up to date with main because this should not show up :)